theano/gpuarray/ROIPoolGPUFwd.c

```diff
@@ -0,0 +1,120 @@
+#section kernels
+
+#kernel ROIPoolGPUFwd_kernel : ssize, *, ssize, ssize, ssize, ssize, ssize, ssize, *, *, * :
+KERNEL void ROIPoolGPUFwd_kernel(
+    const ga_int num_rois, GLOBAL_MEM DTYPE_i0 *bottom_data,
```

Comment:
I have kept it as ga_int and not the traditional ga_size as the comparison (max and min function) raises an error as i'd be doing the comparison of an unsigned int with a signed integer